src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/GeoJsonRepository.cs

```diff
@@ -1,68 +1,68 @@
 #nullable enable
-using System;
-using System.Collections.Generic;
-using System.Linq;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Data.Model.Database;
 using GovUk.Education.ExploreEducationStatistics.Data.Model.Repository.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Repository
 {
     public class GeoJsonRepository : IGeoJsonRepository
```

Comment:
I think this should be renamed `BoundaryDataRepository`.

When this code is deployed and the migration is run, the table is initially empty. I think we might need to test that existing charts are cached and continue to render ok? I'm assuming you are planning to run the script to populate the `BoundaryData` table as soon as you see the table get created?

Reply:
Might be worth adding details about how this will be deployed to the PR description. Seems like you're going to need a faffy deploy ticket too.

To be super safe, it might also be worth ensuring this doesn't get merged into dev with any other major changes so if we do need to roll back we can do it easily.

Reply:
Yep that's right @benoutram - presumably the existing charts will be thoroughly checked during QA, but they seemed to function locally.

I've raised EES-5330 for the faffy deployment.